core/ui/ViewToolbar/new-journal-here.tid

```diff
@@ -5,7 +5,7 @@ description: {{$:/language/Buttons/NewJournalHere/Hint}}
 
 \whitespace trim
 \define journalButtonTags()
-[[$(currentTiddlerTag)$]] $(journalTags)$
+[[$(currentTiddlerTag)$]] $(journalTags)$ $(moreTags)$
```

Comment:
It would be great to get rid of this macro, too. Albeit a bit more complicated than the instances above!